pkg/serverless/invocationlifecycle/constants.go

```diff
@@ -12,5 +12,8 @@ const (
 	// SpanIDHeader is the header containing the spanID
 	SpanIDHeader = "x-datadog-span-id"
 
+	// InvocationErrorHeader : if set to "true", the extension will know that the current invocation has failed
+	InvocationErrorHeader = "x-datadog-invocation-error"
+
```

Comment:
I would update the comments on these constants to specify which requests the headers are included on. For example, the `InvocationErrorHeader` is present on the "end invocation" endpoint.